Readme.md

```diff
@@ -1,9 +1,33 @@
-## Description
+### Description
 
-This is a training and simple (maybe) project for education  
-This is a cli tool for converting temperature from Celsius to Fahrenheit and vise versa
+This is a training and simple (maybe) project for education.  
+This is a cli tool for converting temperature from Celsius to Fahrenheit and vise versa.
 
-## Tasks
-Create a Github repo (public), plan project structure, use MIT license.  
-Need to decide how to run two different conversions with a single script, e.g. how to decide which conversion you have to perform.  
-Ability to use this tool in CLI and interactive mode.
+### Tasks
+- [x] Create a Github repo (public), plan project structure, use MIT license.  
+- [x] Need to decide how to run two different conversions with a single script, e.g. how to decide which conversion you have to perform.  
+- [x] Ability to use this tool in CLI and interactive mode.
+- [x] Checks input in the interactive mode.
+- [x] Create unit tests.
+- [ ] Create issues for help me to develop my skills 
```

Comment:
@OleksiiBulba yes) But not for me. I asked for help from another people. And this task (Create issues) for another people)

Reply:
I think that this case need to take to another place?

Reply:
I asked about this task because all other task represent the result that we want to create: cli tool, interactive mode, unittests. But this task does not do that: it is an intermediate task that will not be seen in the future. All created issues will be resolved, so there is no reason for this task here. But as it is your project, you can leave it as it is if you wish :-)

Reply:
I suggest staying this task for meanwhile future (~1-3 months). In the farther, I will delete it or move to another section.